src/Compilers/VisualBasic/Portable/Syntax/SyntaxNodeFactories.vb

```diff
@@ -15,6 +15,7 @@ Imports Microsoft.CodeAnalysis.VisualBasic.SyntaxFacts
 Imports InternalSyntax = Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
 Imports Microsoft.CodeAnalysis.Syntax
 Imports System.Collections.Immutable
+Imports System.ComponentModel
```

Comment:
Is this needed?

Reply:
Yes, it's for the `EditorBrowsableAttribute`.

---
In reply to: [432677987](https://github.com/dotnet/roslyn/pull/44564#discussion_r432677987) [](ancestors = 432677987)